### PR TITLE
AE->OSO ansible doc merge

### PR DESCRIPTION
**Pull request by aveshagarwal**
Changes to make documentation less specific to OSE or AE and also adds README_AEP.md.

### CODE REVIEW

**Comment by aveshagarwal** [comment]
@detiber @sdodson

File: BUILD.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# openshift-ansible RPM Build instructions
+# ansible RPM Build instructions
```
Comment (wshearn) [inline BUILD.md]: This needs to be changed as it seems a bit ambiguous to the ansible package.

File: docs/best_practices_guide.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // vim: ft=asciidoc
 
-= openshift-ansible Best Practices Guide
+= Ansible Best Practices Guide
```
Comment (wshearn) [inline docs/best_practices_guide.adoc]: Also ambiguous as this refers to best practices for this repo and not ansible as a whole

**Comment by aveshagarwal** [comment]
@wshearn ok will update the PR.

**Comment by aveshagarwal** [comment]
@wshearn  updated the PR.

**Comment by wshearn** [comment]
Ok, looks good to me. @twiest thoughts?

File: docs/best_practices_guide.adoc [hunk 3]
```diff
@@ -421,7 +421,7 @@ For consistency, role names SHOULD follow the above naming pattern. It is import
 Many times the `technology` portion of the pattern will line up with a package name. It is advised that whenever possible, the package name should be used.
 
 .Examples:
-* The role to configure an OpenShift Master is called `openshift_master`
+* The role to configure a Master is called `openshift_master`
```
Comment (detiber) [inline docs/best_practices_guide.adoc]: s/a Master/a master/

nitpicky, but not sure master should be capitalized in this context...  probably shouldn't have been in the first place :)

File: roles/openshift_common/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-OpenShift Common
-================
+OpenShift/Atomic Enterprise Common
+===================================
```
Comment (detiber) [inline roles/openshift_common/README.md]: Now this is role README naming I could get behind...  :+1:

**Comment by aveshagarwal** [comment]
@detiber will update the PR.

**Comment by aveshagarwal** [comment]
@detiber Pushed updated PR.

**Comment by ashcrow** [comment]
@wshearn / @detiber mind taking a look to see if it's ready to merge?

**Comment by detiber** [comment]
:+1:

**Comment by wshearn** [comment]
:+1:

**Comment by ashcrow** [comment]
cc @brenton

**Comment by twiest** [comment]
ok to test

**Comment by twiest** [comment]
:+1:

**Pull request merged**